python/packages/zarathustra/agents/oracle_verifier/aea-config.yaml

```diff
@@ -121,6 +121,9 @@ models:
       - eightballer/oracle_verifier_chronicle
       - eightballer/oracle_verifier_pyth
       - eightballer/oracle_verifier_flare
+      subgraph_api_key: ${GRAPH_API_KEY:str}
+      subgraph_url: 'https://gateway.thegraph.com/api/{api_key}/subgraphs/id/4RTrnxLZ4H8EBdpAQTcVc7LQY9kk85WNLyVzg5iXFQCH'
+      subgraph_query: '{assets {id symbol}}'
```

Comment:
TODO: uniswap subgraph endpoint and query. Currently this is a "random" subgraph and query (of some chainlink price feed)